docs/intro.md

```diff
@@ -3,65 +3,176 @@
-    - Fedora: `sudo dnf install verilator`
-    - Mac: `brew install verilator`
-    - Other platforms: [Verilator installation][verilator-install].
-4. [jq][]:
-    - Ubuntu: `sudo apt install jq`
-    - Fedora: `sudo dnf install jq`
-    - Mac: `brew install jq`
-    - Other platforms: [JQ installation][jq-install].
-5. (Optional) [flit][]:
-    - `python -m pip install flit`
+We use [Verilator][verilator] to simulate compiled designs and verify correctness. If you're on a Mac,
+install with:
+```
+brew install verilator
+```
 
-### Install the FuTIL Driver
+Otherwise, you will probably need to compile it from source yourself (the versions in Linux repositories are generally out-of-date.)
+There instructions are stolen from [Verilator install][verilator-install]:
+```
+git clone https://github.com/verilator/verilator
+cd verilator
+git pull
+git checkout master
+autoconf
+./configure
+make
+sudo make install
+```
 
-[The FuTIL driver](./tools/fud.md) is required to run the various tests.
-Follow the [installation instructions](./tools/fud.html#installation):
+Verilator can produce memory dumps and [VCD][] files (reporting the values of signals at every clock cycle).
+Install `vcdump` so that `fud` can produce JSON representations of the VCD files for easier command-line
+handling.
 ```
-cd fud && flit install -s
+cargo install vcdump
 ```
+</details>
 
-### Install the FuTIL Python Library
+### Python frontends (Systolic array, NTT, MrXL)
+<details><summary>Click to expand!</summary>
 
-[Frontend compilers][frontends] use a [Python library][calyx-py] to emit FuTIL
-programs:
+You need [flit][] to install our python frontends.
+```
+pip3 install flit
+```
+
+Our python [frontends][frontends] use a FuTIL ast library written in python. Install with:
 ```
 cd calyx-py && flit install -s
 ```
 
-### Building and Testing
+Frontend specific instructions:
+ - [Systolic array](./frontends/systolic-array.md):
+ Nothing else needed.
+ - NTT: `pip3 install prettytables`
+ - [MrXL](./frontends/mrxl.md): `cd frontends/mrxl && flit install -s`
 
-In the root of the repository, run the following:
+</details>
 
-1. Run `cargo build` to build the compiler.
-2. Run `./target/debug/futil --help` to get options from the `futil` binary.
-  Alternatively, run `cargo run -- --help` which rebuilds and runs the compiler.
+### Dahlia frontend
+<details><summary>Click to expand!</summary>
 
-3. Run the tests (excluding the test for the frontend compilers):
-```bash
-runt --exclude frontend
+[Dahlia][dahlia] is an imperative HLS language that supports FuTIL as a backend.
+[Here][dahlia-install] are the complete instructions, but we've provided a quick overview.
+First, install [sbt][].
+Then:
+```
+git clone https://github.com/cucapra/dahlia.git
+cd dahlia
+sbt assembly
+```
+
+If you have `fud` installed, tell `fud` where the Dahlia compiler lives:
+```
+fud config stages.dahlia.exec $(pwd)/fuse
+```
+</details>
+
+### Vivado/VivadoHLS Synthesis backends
```

Comment:
Is this strictly necessary for "getting started"? We could make this a "where to go next" question. The Fud installation section on backends already has overlapping information